static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -0,0 +1,261 @@
+import ajax from 'lib/ajax';
+import mediator from 'lib/mediator';
+import checkMediator from 'common/modules/check-mediator';
+import addEventListener from 'lib/add-event-listener';
+import identity from 'common/modules/identity/api';
+import url from 'lib/url';
+import cookies from 'lib/cookies';
+import robust from 'lib/robust';
+import userTiming from 'lib/user-timing';
+import config from 'lib/config';
+import newHeaderNavigation from 'common/modules/navigation/newHeaderNavigation';
+import ga from 'common/modules/analytics/google';
+import debounce from 'lodash/functions/debounce';
+import ophan from 'ophan/ng';
+
+const setAdTestCookie = (): void => {
+    const queryParams = url.getUrlVars();
+
+    if (queryParams.adtest === 'clear') {
+        cookies.remove('adtest');
+    } else if (queryParams.adtest) {
+        cookies.add('adtest', encodeURIComponent(queryParams.adtest), 10);
+    }
+};
+
+const showHiringMessage = (): void => {
+    try {
+        if (!config.page.isDev) {
+            window.console.log(
+                '\n' +
+                    '%cHello.\n' +
+                    '\n' +
+                    '%cWe are hiring – ever thought about joining us? \n' +
+                    '%chttp://developers.theguardian.com/join-the-team.html%c \n' +
+                    '\n',
+                'font-family: Georgia, serif; font-size: 32px; color: #005689',
+                'font-family: Georgia, serif; font-size: 16px; color: #767676',
+                'font-family: Helvetica Neue, sans-serif; font-size: 11px; text-decoration: underline; line-height: 1.2rem; color: #767676',
+                ''
+            );
+        }
+    } catch (e) {
+        /* do nothing */
+    }
+};
+
+const handleMembershipAccess = (): void => {
+    const { membershipUrl, membershipAccess, contentId } = config.page;
+
+    const redirect = (): void => {
+        window.location.href = `${membershipUrl}/membership-content?referringContent=${contentId}&membershipAccess=${membershipAccess}`;
+    };
+
+    if (identity.isUserLoggedIn()) {
+        ajax({
+            url: `${membershipUrl}/user/me`,
+            type: 'json',
+            crossOrigin: true,
+            withCredentials: true,
+        })
+            .then(resp => {
+                const requireClass = 'has-membership-access-requirement';
+                const requiresPaidTier = !membershipAccess.includes(
+                    'paid-members-only'
+                );
+                // Check the users access matches the content
+                const canViewContent = requiresPaidTier
+                    ? !!resp.tier && resp.isPaidTier
+                    : !!resp.tier;
+
+                if (canViewContent) {
+                    const body = document.body;
+
+                    fastdom.write(() => {
+                        if (body) {
+                            body.classList.remove(requireClass);
+                        }
+                    });
+                } else {
```

Comment:
Interesting! `finally()` is at Stage 2 at the moment, so this shouldn't be in there. But yeah - if pf.io provides it, why noi/t?! 👍